src/studio/src/designer/frontend/ux-editor/App.tsx

```diff
@@ -30,12 +27,10 @@ export function App() {
       dispatch(FormLayoutActions.fetchFormLayout());
 
       const languageCode = 'nb';
-      dispatch(
-        loadTextResources({url: getLoadTextResourcesUrl(languageCode)}),
-      );
+      dispatch(loadTextResources({ url: getLoadTextResourcesUrl(languageCode) }));
       dispatch(fetchServiceConfiguration());
       dispatch(fetchRuleModel());
-      dispatch(fetchLanguage({languageCode}));
+      dispatch(fetchLanguage({ languageCode }));
```

Comment:
This could be changed to be consistent with the rest of the applications, which use a `url` parameter and no language code

Reply:
As we probably will move to https://messageformat.github.io/messageformat/ I don't think this is an issue for long. All lanuage support should be consolidated and done the "messageformat-way".